packages/adaptive-ui-explorer/src/components/style-example.ts

```diff
@@ -15,14 +15,16 @@ const template = html<StyleExample>`
                 <slot></slot>
             </app-adaptive-component>
         </div>
-        ${repeat(x => x.styleValues, html<StyleValue, StyleExample>`
-            <app-swatch
-                type="${x => x.type}"
-                recipe-name="${x => x.tokenName}"
-                :fillRecipe="${x => x.fillRecipe}"
-                :foregroundRecipe="${x => x.foregroundRecipe}"
-                :outlineRecipe="${x => x.outlineRecipe}"
-            ></app-swatch>
+        ${when(x => x.showSwatches, html<StyleExample>`
+            ${repeat(x => x.styleValues, html<StyleValue, StyleExample>`
```

Comment:
A small perf improvement could be to extract these inline templates in their own variables outside of the larger template that composes them. This would make it so the templates get constructed once and only the bound behaviors are what changes.